prime-router/settings/prod/0037-vt-fix.yml

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
Per @mauricereeves , added a ordering_provider_id requirement for VT-DOH.
This is already deployed to Prod.